docs/walkthrough/kremling-inline-loader.md

```diff
@@ -0,0 +1,54 @@
+# Kremling inline loader
+
+[https://github.com/CanopyTax/kremling-loader](https://github.com/CanopyTax/kremling-loader)
```

Comment:
what is this link for again?

Reply:
kremling-inline-loader live in the kremling-loader repo